sharding/nodesSetup_test.go

```diff
@@ -37,20 +57,12 @@ func createNodesSetupTwoShardTwoNodes() *sharding.NodesSetup {
 	ns.ConsensusGroupSize = 1
 	ns.MinNodesPerShard = 2
 	ns.InitialNodes = make([]*sharding.InitialNode, 4)
```

Comment:
I would create a variable for initial nodes number, and use that in the memory allocation and loop and anywhere else it is needed, instead of using same magic value in multiple places, then it would be easy to change.

Applies to other tests in this file.

Reply:
Done